migrations/1605637330716-CreateMakerBalanceChainCache.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+const makerBalanceChainCacheTable = new Table({
+    name: 'maker_balance_chain_cache',
+    columns: [
+        { name: 'token_address', type: 'varchar', isPrimary: true },
+        { name: 'maker_address', type: 'varchar', isPrimary: true },
+        // fields are nullable for the job that adds new rows upon
+        // discovery of a new maker address
+        { name: 'balance', type: 'varchar', isNullable: true },
+        { name: 'time_of_sample', type: 'timestamptz', isNullable: true },
```

Comment:
@alexkroeger just curious, should we also add a `time_first_seen` column (that is written the first time a row is added to this table by the API?

Reply:
Yea I like that 👍